app/package/controllers.py

```diff
@@ -82,8 +82,6 @@ def tag_data_package(publisher, package):
         status_db = Package.create_or_update_tag(publisher, package, data['version'])
         status_bitstore = bitstore.copy_to_new_version(data['version'])
 
-        if status_db is False or status_bitstore is False:
-            raise Exception("failed to tag data package")
         return jsonify({"status": "OK"}), 200
     except Exception as e:
```

Comment:
Why are we catching the exception here? Why can't we do this in a generic top level handler?